frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingLegaleseModal/EmbeddingLegaleseModal.tsx

```diff
@@ -1,19 +1,30 @@
 import { useState } from "react";
+import { match } from "ts-pattern";
 import { t } from "ttag";
 
 import { useUpdateSettingsMutation } from "metabase/api";
 import { Button, Group, List, Modal, type ModalProps, Text } from "metabase/ui";
 
-export const EmbeddingSdkLegaleseModal = ({ opened, onClose }: ModalProps) => {
+type SettingKey = "enable-embedding-sdk" | "enable-embedding-simple";
+
+export const EmbeddingLegaleseModal = ({
```

Comment:
We won't be using `EmbeddingLegaleseModal` in this PR, but it will be used in the next PR on setting detail page.

I was gonna use this in this PR because we were going to have two cards, but now that we have one unified card this toggle would be shown in the setting detail page (Configure) instead.